tests/unit/lib/telemetry/test_event.py

```diff
@@ -93,3 +93,83 @@ def test_track_event(self, event_mock, lock_mock):
         self.assertEqual(len(EventTracker._events), 0)
         lock_mock.__enter__.assert_called()  # Lock should have been accessed
         lock_mock.__exit__.assert_called()
+
+    @patch("samcli.lib.telemetry.event.Telemetry")
+    def test_events_get_sent(self, telemetry_mock):
+        # Create fake emit to capture tracked events
+        dummy_telemetry = Mock()
+        tracked_events = []
```

Comment:
nit: could we rename to `emitted_metrics`? (if I understand correctly)

Reply:
`emitted_events` may be more accurate, but yes that would probably be more specific and correct

Reply:
but doesn't `emit` get called with a `Metric`? 🤔 
Maybe `emitted_event_metrics`? lol